pkgs/top-level/all-packages.nix

```
@@ -25948,6 +25948,9 @@ in
 
   hll2390dw-cups = callPackage ../misc/cups/drivers/hll2390dw-cups { };
 
+  mfc9332cdwcupswrapper = (callPackage ../misc/cups/drivers/brother/mfc9332cdw { }).mfc9332cdwcupswrapper;
```

```suggestion
  mfc9332cdw-cupswrapper = (callPackage ../misc/cups/drivers/brother/mfc9332cdw { }).cupswrapper;
```

to match the attribute name below